libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -170,7 +170,7 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           actionCard: {
             description: statesMessages.otherParentApprovalDescription,
           },
-          lifecycle: DEPRECATED_DefaultStateLifeCycle,
```

Comment:
The application has to be available in at least 2 years and 8 months.
Is it? pruneAfterDays(970) // available for 2 years and 8 months